[new-product]: fluent-bit (#5015)
captn3m0/endoflife.date: products/fluent-bit.md

---

> title: Fast and Lightweight Log processor and forwarder

Reviewer: Title is name of the product.

```suggestion
title: Fluent Bit
```

> category: app

Reviewer: ```suggestion
category: server-app
```

> tags: kubernetes

Reviewer: We're using cncf for most products, and reserving the kubernetes tag for products that are very specific to k8s.

```suggestion
tags: cncf
```

permalink: /fluent-bit
alternate_urls:
-   /fluentbit
versionCommand: |-
  fluent-bit -v
releasePolicyLink: https://fluentbit.io/announcements/older-versions
changelogTemplate: https://github.com/fluent/fluent-bit/releases/tag/v__LATEST__
releaseDateColumn: true
eolColumn: Support

auto:
  methods:
  -   git: https://github.com/fluent/fluent-bit

identifiers:

> -   purl: pkg:github/fluent/fluent-bit
> -   purl: pkg:docker/fluent/fluent-bit

Reviewer: This will add a link to the various distro packages: https://repology.org/project/fluent-bit/versions

```suggestion
-   purl: pkg:github/fluent/fluent-bit
-   purl: pkg:docker/fluent/fluent-bit
-   repology: fluent-bit
```

releases:
-   releaseCycle: "3.0"
    releaseDate: 2024-03-21
    eol: false
    latest: "3.0.2"
    latestReleaseDate: 2024-04-12

-   releaseCycle: "2.2"
    releaseDate: 2023-11-06
    eol: false
    latest: "2.2.2"
    latestReleaseDate: 2023-12-22

-   releaseCycle: "2.1"
    releaseDate: 2023-04-19

>     eol: true

Reviewer: We prefer using dates wherever available.

```suggestion
    eol: 2024-02-06
```

    latest: "2.1.10"
    latestReleaseDate: 2023-10-28

-   releaseCycle: "2.0"
    releaseDate: 2022-10-25

>     eol: true

Reviewer: ```suggestion
    eol: 2023-07-19
```

    latest: "2.0.9"
    latestReleaseDate: 2023-02-06

-   releaseCycle: "1.9"
    releaseDate: 2022-03-11

>     eol: true

Reviewer: ```suggestion
    eol: 2022-12-05
```

    latest: "1.9.9"
    latestReleaseDate: 2022-09-29

-   releaseCycle: "1.8"
    releaseDate: 2021-07-07

>     eol: true

Reviewer: ```suggestion
    eol: 2022-12-05
```

    eol: true
    latest: "1.4.6"
    latestReleaseDate: 2020-06-10

-   releaseCycle: "1.3"
    releaseDate: 2019-09-30
    eol: true
    latest: "1.3.11"
    latestReleaseDate: 2020-03-19

-   releaseCycle: "1.2"
    releaseDate: 2019-06-27
    eol: true
    latest: "1.2.2"
    latestReleaseDate: 2019-07-19

-   releaseCycle: "1.1"
    releaseDate: 2019-05-09
    eol: true
    latest: "1.1.3"
    latestReleaseDate: 2019-06-13

-   releaseCycle: "1.0"
    releaseDate: 2018-12-13
    eol: true
    latest: "1.0.6"
    latestReleaseDate: 2019-03-27

-   releaseCycle: "0.14"
    releaseDate: 2018-08-29
    eol: true
    latest: "0.14.9"
    latestReleaseDate: 2018-12-07

-   releaseCycle: "0.13"
    releaseDate: 2018-08-18
    eol: true
    latest: "0.13.8"
    latestReleaseDate: 2018-04-30

-   releaseCycle: "0.12"
    releaseDate: 2017-08-17
    eol: true
    latest: "0.12.19"
    latestReleaseDate: 2018-04-22

-   releaseCycle: "0.11"
    releaseDate: 2017-03-30
    eol: true
    latest: "0.11.17"
    latestReleaseDate: 2017-08-15

-   releaseCycle: "0.10"
    releaseDate: 2016-12-30
    eol: true
    latest: "0.10.1"
    latestReleaseDate: 2017-01-06

---

> > The fluent-bit is a Fast and Lightweight Logs and Metrics processor for Linux, BSD, OSX and Windows.

Reviewer: ```suggestion
> Fluent Bit is a cross-platform fast and lightweight logs and metrics processor. It is an
> open-source CNCF sub-project under the umbrella of [Fluentd](https://www.fluentd.org).
```

Support for a series (x.y) ends three months after a new series is released.
